kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -37,23 +41,44 @@
   const constants = require('../../state/constants');
   const getters = require('../../state/getters');
   const ContentKinds = require('kolibri.coreVue.vuex.constants').ContentKinds;
+  const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
     $trNameSpace: 'learnContent',
     $trs: {
       recommended: 'Recommended',
+      nextContent: 'Next Content',
     },
     computed: {
-      canDownload() { return this.kind !== ContentKinds.EXERCISE; },
+      canDownload() { return this.content.kind !== ContentKinds.EXERCISE; },
+      showNextBtn() { return this.content.kind === ContentKinds.EXERCISE; },
       recommendedText() {
         return this.$tr('recommended');
       },
+      progress() {
+        return this.userkind.includes(
+          UserKinds.LEARNER) ? this.summaryProgress : this.sessionProgress;
+      },
     },
     mixins: [constants], // makes constants available in $options
     components: {
       'page-header': require('../page-header'),
       'expandable-content-grid': require('../expandable-content-grid'),
     },
+    methods: {
+      MoveToNextContent() {
```

Comment:
Should start with lower-case. This is also returning a link, not performing an action, so it should be a computed property - not a method.

Maybe a better name is `nextContentLink`?